src/test/java/uk/gov/hmcts/reform/juddata/camel/helper/JrdTestSupport.java

```diff
@@ -15,8 +26,15 @@
 import uk.gov.hmcts.reform.juddata.camel.binder.JudicialUserRoleType;
 import uk.gov.hmcts.reform.juddata.camel.route.beans.RouteProperties;
 
+@TestPropertySource(properties = {"spring.config.location=classpath:application-integration.yml"})
 public class JrdTestSupport {
 
+    @Value("${route.judicial-user-profile-orchestration.route-header}")
+    public static String fileHeader;
+
+    public static Map readYmlAsMap = readYmlAsMap("test-header.yaml");
```

Comment:
do you need to maintain separate yaml?

Reply:
its for testing purpose only and  hardcoding header will looks messy